pkg/controller/ibmblockcsi/syncer/csi_controller.go

```diff
@@ -157,15 +155,64 @@ func (s *csiControllerSyncer) ensureContainersSpec() []corev1.Container {
 	}
 }
 
+func ensureDefaultResources() corev1.ResourceRequirements {
+	return ensureResources("50m", "100m", "50Mi", "100Mi")
+}
+
+func ensureResources(cpuRequests, cpuLimits, memoryRequests, memoryLimits string) corev1.ResourceRequirements {
+	requests := corev1.ResourceList{
+		corev1.ResourceRequestsCPU:    resource.MustParse(cpuRequests),
+		corev1.ResourceRequestsMemory: resource.MustParse(memoryRequests),
+	}
+	limits := corev1.ResourceList{
+		corev1.ResourceLimitsCPU:    resource.MustParse(cpuLimits),
+		corev1.ResourceLimitsMemory: resource.MustParse(memoryLimits),
+	}
+
+	return corev1.ResourceRequirements{
+		Limits:   limits,
+		Requests: requests,
+	}
+}
+
+func ensureNodeAffinity() *corev1.NodeAffinity {
+	return &corev1.NodeAffinity{
+		RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+			NodeSelectorTerms: []corev1.NodeSelectorTerm{
+				{
+					MatchExpressions: []corev1.NodeSelectorRequirement{
+						{
+							Key:      "beta.kubernetes.io/arch",
```

Comment:
please fix the key to kubenetes.io/arch (with no beta prefix - i think its deprecated also from k8s 1.14)